lib/medicalInformation/schema.graphql

```diff
@@ -0,0 +1,12 @@
+type MedicalInformation {
```

Comment:
Do we also need to be adding resolvers for the `MedicalInformation` and `Guardian`?

Reply:
Not at the top level. However, we would want to have field resolvers for medical information and guardian so that they can be retrieved when querying for applicants. I mentioned this in my comment above.